proto/src/determined/experiment/v1/searcher.proto

```diff
@@ -61,25 +53,27 @@ message TrialExitedEarly {
     EXITED_REASON_INIT_INVALID_HP = 3;
   }
   // The reason for the exit.
-  ExitedReason exited_reason = 3;
+  ExitedReason exited_reason = 2;
 }
 
 // SearcherEvent is a message from master to a client-driven
 // custom searcher informing it of relevant changes in the state
 // of an experiment
 message SearcherEvent {
+  // incremental id of the event
+  int32 id = 1;
   // The concrete event
   oneof event {
     // an experiment has just been created
-    InitialOperations initial_operations = 1;
+    InitialOpsEvent initial_ops_event = 3;
```

Comment:
I know I okayed this but I think we should be consistent: either we add Event suffix to all events or we don't. I don't care much which way we go.

Reply:
I might suffix to all of them. Will think about if there's better way.